website/src/models/user.js

```diff
@@ -390,7 +390,14 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
-    }
+    },
+    improvementCategories: {
```

Comment:
This'll actually be nice for improved onboarding on the Web site later, too! :+1: